python/cugraph/jaccard/wjaccard_wrapper.pyx

```diff
@@ -74,29 +74,29 @@ cpdef jaccard_w(input_graph, weights, first=None, second=None):
     """
 
     cdef uintptr_t graph = input_graph.graph_ptr
-    cdef gdf_graph * g = < gdf_graph *> graph
+    cdef gdf_graph * g = <gdf_graph*> graph
     
     err = gdf_add_adj_list(g)
     cudf.bindings.cudf_cpp.check_gdf_error(err)
 
-    cdef uintptr_t result_ptr
-    cdef uintptr_t weight_ptr
-    cdef uintptr_t first_ptr
-    cdef uintptr_t second_ptr
-    cdef uintptr_t indices_ptr
+    cdef gdf_column c_result_col
+    cdef gdf_column c_weight_col
+    cdef gdf_column c_first_col
+    cdef gdf_column c_second_col
+    cdef gdf_column c_indices_col
```

Comment:
Should this be c_index_col

Reply:
Thanks for finding this!!! and I fixed this.